Assets/Scripts/Importing/GTX/GXT.cs

```diff
@@ -7,34 +7,35 @@
 
 namespace SanAndreasUnity.Importing.GXT
 {
-    public class GXT : IDisposable
-    {
-
-	    #region LoadPath
-		//this whole region needs refactoring
-	    enum Language
-	    {
-		    american,
-		    french,
-		    german,
-		    italian,
-		    spanish
-	    }
-
-
-	    private static Language defaultLanguage = Language.american;
-	    //this code is pasted from ArchiveMgr; 
-	    //todo will refactor in anther commit
-	    public static string GameDir => Config.Get<string>("game_dir");
-	    public static string GTXDir => Path.Combine(GameDir, "text");
-
-	    //todo put path temp here, will refactor
-	    private static string loadPath=> Path.Combine(GTXDir,$"{defaultLanguage}.gxt");
+	public class GXT : IDisposable
+	{
+
+		#region LoadPath
+		private static string _language= "american";
+		public static string Language
+		{
+			get => _language;
+			set
+			{
+				var path = GetPathFromLanguage(value);
+				if (File.Exists(path))
+				{
+					_language = value;
+				}
+			}
+		}
+
+		//this code is pasted from ArchiveMgr; 
+		public static string GameDir => Config.Get<string>("game_dir"); //todo this should be put in a config 

```

Comment:
GameDir: just use `Config.GamePath` for this

Reply:
Yes. Just found it today. Will fix it.